rust/worker/src/blockstore/positional_posting_list_value.rs

```diff
@@ -50,16 +48,6 @@ pub(crate) enum PositionalPostingListBuilderError {
     UnsortedPosition,
 }
 
-impl ChromaError for PositionalPostingListBuilderError {
```

Comment:
I might be missing context here - I don't understand the reason for removing these? I think we should still have error codes for these enums. The caller can propagate it upstream by using `e.code()` for instance

Reply:
Good point, added it back